chromadb/test/property/test_embeddings.py

```diff
@@ -128,21 +137,49 @@ def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID
             }
             self.collection.add(**normalized_record_set)
             self._upsert_embeddings(cast(strategies.RecordSet, filtered_record_set))
+
```

Comment:
i think initial version has to be after this right?

Reply:
Changed the approach so this comment is moot now